File: src/main/java/com/example/app/repository/SaltRepository.java
package com.example.app.repository;

import com.example.app.utils.model.entities.Salt;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SaltRepository extends CrudRepository<Salt, Long> {
    Salt getSaltByOwner(String owner);
}

File: src/main/kotlin/ru/kaplaan/repository/SaltRepository.kt
package ru.kaplaan.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import ru.kaplaan.domain.entity.Salt

@Repository
interface SaltRepository : CrudRepository<Salt, Long?> {
    fun getSaltByOwner(owner: String): Salt?
}
